_LowLevelCode/cpp/test/combine_sqw.tests/combine_sqw.test.cpp

```diff
@@ -0,0 +1,193 @@
+#include "combine_sqw/combine_sqw.h"
+#include "combine_sqw/nsqw_pix_reader.h"
+#include "combine_sqw/pix_mem_map.h"
+#include "combine_sqw/sqw_pix_writer.h"
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+/*
+ * Retrieve an environment variable from the system.
+ * This function is necessary as `std::getenv` raises security warnings on
+ * Windows.
+ *
+ * @param key :: The key of the environment variable
+ */
+std::string get_enviornment_var(const std::string &key) {
```

Comment:
this should be defined outside this file -- it's a env_helper utility file that will be potentially used by every test file